Gemfile.lock

```diff
@@ -118,6 +118,8 @@ GEM
     minitest (5.14.4)
     msgpack (1.4.2)
     nio4r (2.5.7)
+    nokogiri (1.11.7-arm64-darwin)
```

Comment:
Try to open the terminal using Rosetta (https://apple.stackexchange.com/questions/423915/how-am-i-running-x86-64-programs-without-arch-on-apple-silicon). Afterwards you shouldn't need the changes in this file